eb_hooks.py

```diff
@@ -126,9 +127,15 @@ def post_ready_hook(self, *args, **kwargs):
     Post-ready hook: limit parallellism for selected builds based on software name and CPU target.
                      parallelism needs to be limited because some builds require a lot of memory per used core.
     """
-    # 'parallel' easyconfig parameter is set via EasyBlock.set_parallel in ready step based on available cores.
+    # 'parallel' (EB4) or 'max_parallel' (EB5) easyconfig parameter is set via EasyBlock.set_parallel in ready step
+    # based on available cores.
+
+    # Check whether we have EasyBuild 4 or 5
+    parallel_param = 'parallel'
+    if EASYBUILD_VERSION >= '5':
+        parallel_param = 'max_parallel'
     # get current parallelism setting
-    parallel = self.cfg['parallel']
+    parallel = self.cfg[parallel_param]
```

Comment:
There is also a parallel property in EB5, I'm not sure what the right option is here...

Reply:
@boegel says  `max_parallel` should suffice